Cyral/Cyral.json

```diff
@@ -0,0 +1,1889 @@
+{
+    "type": "FolderSyncDefinition",
+    "name": "Cyral",
+    "description": "",
+    "children": [
+        {
+            "type": "DashboardV2SyncDefinition",
+            "name": "Cyral - Application Activity Details",
+            "description": "Dashboard that provides specific information regarding an application.",
+            "title": "Cyral - Application Activity Details",
+            "rootPanel": null,
+            "theme": "Dark",
+            "topologyLabelMap": {
+                "data": {}
+            },
+            "refreshInterval": 0,
+            "timeRange": {
+                "type": "BeginBoundedTimeRange",
+                "from": {
+                    "type": "RelativeTimeRangeBoundary",
+                    "relativeTime": "-15m"
```

Comment:
All the panels have 15 min relative time Is this intentional? Normally our dashboards have last 24h or last 1h relative time range.

Reply:
This was not intentional. I think it was just the default when I was creating the dashboards. I set everything to 24h